lib/features/family/features/missions/presentation/widgets/mission_widget.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app/features/family/features/missions/presentation/models/mission_ui_model.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/shared/widgets/goal_progress_bar/goal_progress_bar.dart';
+import 'package:givt_app/utils/app_theme.dart';
+
+class MissionWidget extends StatelessWidget {
```

Comment:
This is the GoalCard (inPRogress state), right? Maybe its good to name in the same way sa the design system :)

Reply:
The card in the different states should also already exist for creating/showing family goal. Can we use same component everywhere? :)